lib/SIL/Verifier/SILVerifier.cpp

```diff
@@ -1059,7 +1059,13 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
     
     auto objectTy = value->getType().unwrapOptionalType();
     
-    require(objectTy.isReferenceCounted(F.getModule()),
+    // C++ foreign reference types are represented as trivially lowered types
```

Comment:
Nit: 
```suggestion
    // Immortal C++ foreign reference types are represented as trivially lowered types
```

(because this isn't true for `SWIFT_SHARED_REFERENCE` types, right?)